llvm/test/CodeGen/RISCV/rvv/rvv-peephole-vmerge-vops.ll

```diff
@@ -1192,11 +1192,8 @@ define <vscale x 2 x i32> @vmerge_larger_vl_false_becomes_tail(<vscale x 2 x i32
 define <vscale x 2 x i64> @vpmerge_vwsub.w_tied(<vscale x 2 x i64> %passthru, <vscale x 2 x i64> %x, <vscale x 2 x i32> %y, <vscale x 2 x i1> %mask, i32 zeroext %vl) {
```

Comment:
Might not related to this patch but is the `%x` necessary here?

Reply:
Woops, it's not. Will remove it after this PR!